src/main/config/secor.common.properties

```diff
@@ -139,6 +139,14 @@ kafka.seed.broker.port=9092
 # to as the chroot.
 kafka.zookeeper.path=/
 
+# Store offset in zookeeper and kafka consumer topic.
+# Possible values: true or false
+kafka.dual.commit.enabled=true
+
+# Storage offset.
+# Possible values: "zookeeper" to read offset from zookeeper or "kafka" to read offset from kafka consumer topic
+kafka.offsets.storage=zookeeper
+
```

Comment:
I am not sure about that either, if you got some time, you can give it a try for that parameter, otherwise you can ignore my comment.